crossdock/server/server.go

```diff
@@ -46,10 +48,11 @@ type Server struct {
 	Tracer           opentracing.Tracer
 	listener         net.Listener
 	channel          *tchannel.Channel
+	eHandler         *endtoend.Handler
 }
 
 // Start starts the test server called by the Client and other upstream servers.
-func (s *Server) Start() error {
+func (s *Server) Start(endToEndCfg config.Configuration) error {
```

Comment:
in docker-compose you can easily set env vars

https://github.com/yarpc/yarpc-python/blob/master/docker-compose.yml#L44